### PR TITLE
Inf 371 community release

### PR DESCRIPTION
**Pull request by AliaksandrRyzhou**
A separate workflow to build Community Docker images that should be a multi architecture.

### CODE REVIEW

File: .github/workflows/community-releases.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      packages: write
+      deployments: write
+    steps:
+      - uses: actions/checkout@v4
+
+  docker_build_api:
+    runs-on: ubuntu-latest
+    needs: prepare
+    timeout-minutes: 80
+    environment: Development
```
Comment (LetItRock) [inline .github/workflows/community-releases.yml]: Production

File: .github/workflows/community-releases.yml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        name: ['novu/api-ee', 'novu/api']
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+      - uses: ./.github/actions/setup-project
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+
```
Comment (LetItRock) [inline .github/workflows/community-releases.yml]: for the community release, we shouldn't use the matrix and build the enterprise packages (submodules)

File: .github/workflows/community-releases.yml [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+      - uses: ./.github/actions/docker/build-api
+        id: docker_build
+        with:
+          tag: ${{ inputs.version }}
+          push: 'true'
+          github_token: ${{ secrets.GH_PACKAGES }}
+          docker_name: ${{ matrix.name }}
+          bullmq_secret: ${{ secrets.BULL_MQ_PRO_NPM_TOKEN }}
```
Comment (LetItRock) [inline .github/workflows/community-releases.yml]: this token shouldn't be passed

File: .github/workflows/community-releases.yml [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+          environment: dev
```
Comment (LetItRock) [inline .github/workflows/community-releases.yml]: prod?

File: .github/workflows/community-releases.yml [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+          community_build: true
+
+  docker_build_worker:
```
Comment (LetItRock) [inline .github/workflows/community-releases.yml]: similar comments to this and other jobs

File: .github/workflows/reusable-docker.yml [hunk 6]
```diff
@@ -35,6 +35,10 @@ on:
       env_tag:
         required: false
         type: string
+      community_build:
+        required: false
+        default: false
+        type: boolean
```
Comment (LetItRock) [inline .github/workflows/reusable-docker.yml:41]: this file should be adjusted based on this argument, the same thing, we don't need to build the enterprise packages

File: .github/workflows/community-releases.yml [hunk 7]
```diff
@@ -0,0 +1,160 @@
+name: Docker Community Release
```
Comment (SokratisVidros) [inline .github/workflows/community-releases.yml:1]:
```suggestion
name: "Prepare Community Release" to match with "Prepare Cloud Release"
```

File: .github/workflows/community-releases.yml (hunk 2 above)
Comment (SokratisVidros) [reply .github/workflows/community-releases.yml]: @LetItRock shouldn't we include it in the workflow inputs for self-hosted EE deployments?

**Review by SokratisVidros — CHANGES_REQUESTED** [review]

File: .github/workflows/community-releases.yml (hunk 2 above)
Comment (Cliftonz) [reply .github/workflows/community-releases.yml:44]: @SokratisVidros @LetItRock From working with the enterprise clients when we release a community version would should also release a enterprise version.

**Comment by Cliftonz** [comment]
@SokratisVidros Do you want the ee builds with the community or separate?

**Pull request closed without merging**